### PR TITLE
Host videos on Vimeo instead of YouTube

### LINKED ISSUE #1181: Switch YouTube videos to Vimeo
YouTube now requires to show related videos after a video has finished, which is not useful for educational purposes. We should switch our videos to Vimeo, as we have done for our other projects.

### PR DESCRIPTION
**Pull request by courtneycb**
Resolves #1181. Note that this only replaces UCCSER owned YouTube videos.

Videos can be found under the 'CS Unplugged' folder on Vimeo.

### CODE REVIEW

**Comment by courtneycb** [comment]
@eAlasdair ready for re-review

**Comment by eAlasdair** [comment]
There's an option of subtitles now but they don't show on the site nor on Vimeo itself.

Nice work on the pause though

**Comment by courtneycb** [comment]
> There's an option of subtitles now but they don't show on the site nor on Vimeo itself.
> Also would probably be best if subtitles were enabled by default - particularly on non-english versions even if the right language isn't available yet

I don't get this problem?
image
image

**Comment by courtneycb** [comment]
> > I don't get this problem?
> 
> I still do, chrome & FF, as well as chrome on my laptop
> 
> Is it because you're logged into vimeo maybe?

Can you come in and show us in the office? Jack isn't getting the problem either.

File: subtitles/en/parity-magic-trick.vtt [hunk 1]
```diff
@@ -11,7 +11,7 @@ Date: 13/02/2017
```
Comment (eAlasdair) [inline subtitles/en/parity-magic-trick.vtt:11]: Maybe say that you edited this file, though the translators didn't say anything either

**Pull request merged**